src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,177 @@
 package co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
+import java.util.function.Function;
 
-public class Range<T> implements Set<T> {
+public class Range<T> implements Set<T> , Comparable {
+
+    static Object[] elementData = new Object[10];
```

Comment:
We do not know at the beginning how many items will be. What will happen if I add the eleventh element? You have to handle it.